docs/current_docs/manuals/user/artifacts/consumption/273920-export.mdx

```diff
@@ -0,0 +1,49 @@
+---
+slug: /manuals/user/273920/export
+---
+
+# Artifact Export
+
+Just-in-time artifacts such as containers, directories and files expose an `Export()` function, enabling them to be exported to the host filesystem from the Dagger Function that produced them. The destination path on the host is passed to the function using the `--path` argument.
+
+Here is an example of exporting the directory returned by a Go builder Dagger Function to the `./my-build` directory on the host:
+
+```shell
+dagger -m github.com/kpenfound/dagger-modules/golang@v0.1.5 call build --project=https://github.com/dagger/dagger --args=./cmd/dagger export --path=./my-build
```

Comment:
Updated: `dagger -m github.com/kpenfound/dagger-modules/golang@v0.1.10 call build --source=https://github.com/dagger/dagger --args=./cmd/dagger export
 --path=./my-dagger-build`
 
 Notice the argument is `source` instead of `project` for the newer version

Reply:
Modified